ordering/ordering.proto

```diff
@@ -3,12 +3,10 @@ syntax = "proto3";
 package ordering;
 option go_package = "github.com/relab/gorums/ordering";
 
-import "google/protobuf/any.proto";
-
 service Gorums { rpc NodeStream(stream Message) returns (stream Message); }
 
 message Message {
   uint64 ID = 1;
-  string Method = 2;
-  google.protobuf.Any Data = 3;
+  uint32 MethodID = 2;
```

Comment:
Why not make this `int32` to avoid all the type conversion noise.

Reply:
Yeah, I'll change them all to the same type.